test/health_gateway/plugins/status/server/plugin.ts

```diff
@@ -13,23 +13,27 @@ export class HealthGatewayStatusPlugin implements Plugin<void, void> {
   public setup(core: CoreSetup) {
     const router = core.http.createRouter();
 
-    router.get({ path: '/api/status/ok', validate: {} }, async (context, req, res) => res.ok());
+    router.get({ path: '/health/ok/api/status', validate: {} }, async (context, req, res) =>
+      res.ok()
+    );
```

Comment:
We were working around the configuration in our FTR tests by specifying specific urls of our plugin for each of the test "instances" that the gateway was reaching. Given we now add `/api/status` to the host's url, I had to slightly change the urls of the FTR test plugin and associated config files.